.eslintrc.js

```diff
@@ -40,16 +40,27 @@ module.exports = {
     "no-unused-vars": ["error", { args: "none" }],
     "no-with": "error",
     "object-curly-spacing": "off",
+    "react/button-has-type": "error",
     "react/display-name": "off",
     "react/jsx-closing-bracket-location": "error", // autofixable
     "react/jsx-curly-spacing": "error", // autofixable
     "react/jsx-first-prop-new-line": ["error", "multiline"],
     "react/jsx-indent-props": ["error", 2], // autofixable
+    "react/jsx-key": "off",
+    "react/jsx-no-target-blank": "off", // TODO: turn this on
```

Comment:
Task created for this TODO?

Reply:
Hm... I'll tear out the TODO.  It seemed like a good one to turn on at a later date, but not sure it's worth tracking.

Reply:
Is there a reason _not_ to have this turned on? It's protecting against a legit security vulnerability

Reply:
I'd actually be interested in trying to tun on _most_ of these disabled rules. Maybe we could add a task to track that larger goal?

Reply:
:+1: I'm into that - I don't mind strict linting.  I'm going to tear out all TODOs in this file though, and we'll track this work elsewhere.

Reply:
> Is there a reason _not_ to have this turned on? It's protecting against a legit security vulnerability

:cry: Only that we violate this rule all over the place.  Propose turning this on and fixing in a follow-up PR?

Reply:
https://codedotorg.atlassian.net/browse/XTEAM-139